CustomPlayerModels/src/shared/java/com/tom/cpl/gui/elements/Spinner.java

```diff
@@ -52,31 +55,54 @@ public void draw(MouseEvent event, float partialTicks) {
 		if(txtf.isFocused() && error != null) {
 			gui.drawRectangle(bounds.x, bounds.y, bounds.w, bounds.h, 0xffff0000);
 		}
+		// mouse click repeat
+		if (currentClick != null && MinecraftClientAccess.get().getPlayerRenderManager().getAnimationEngine().getTicks() - currentClickStartTime > 10) {
+				arrowClicked(currentClick);
```

Comment:
Add `currentClickStartTime++;` so it'll increment 20 times a second, not every frame.

Reply:
Done.